opal/mca/threads/thread_usage.h

```diff
@@ -0,0 +1,193 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
```

Comment:
TODO: @npe9 need to actually move from opal/threads/thread_usage.h down to the framework